src/cf-typography/usage.md

```diff
@@ -7,30 +7,25 @@ and has more basic typography patterns.
 ## Contents
 
 - [Variables](#variables)
-- [Pull quote](#pull-quote)
-- [Micro copy](#micro-copy)
-- [Short description](#short-description)
-- [Date](#date)
+  - [Color variables](#color-variables)
+- [Running text](#running-text)
```

Comment:
What should the group be? These are kind of on their own since they aren't headings, headers, or links.